src/helpers.js

```diff
@@ -469,9 +467,7 @@ export function createStreamInfo(item, mediaSource, startPosition) {
         console.log('Subtitle url: ' + info.subtitleStreamUrl);
     });
 
-    if (subtitleTracks) {
-        info.tracks = subtitleTracks;
-    }
+    info.tracks = subtitleTracks;
```

Comment:
My conclusion is that `tracks` refer to text tracks and that an empty array here doesn't hurt.

The previous if-check always returns true, so this should have no impact.

https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.MediaInformation